src/kitovu/sync/plugin/smb.py

```diff
@@ -154,7 +173,12 @@ def retrieve_file(self,
                       path: pathlib.PurePath,
                       fileobj: typing.IO[bytes]) -> typing.Optional[int]:
         logger.debug(f'Retrieving file {path}')
-        self._connection.retrieveFile(self._info.share, str(path), fileobj)
+        try:
+            self._connection.retrieveFile(self._info.share, str(path), fileobj)
+        except OperationFailure:
+            raise utils.PluginOperationError(
+                f'Could not download {path} from share "{self._info.share}"')
```

Comment:
Not sure if we should handle this - this is something which should never happen, right?